### DISCUSSION TITLE
Coding style

### DISCUSSION THREAD

**Discussion by platinops** [elimintz/justpy#490] [General]
I'm happy to see that justpy is again being actively developed.

I did note however that several commits to not adhere to [PEP8](https://peps.python.org/pep-0008/). Some examples:

- `getServer` -> `get_server` 
(incorrect use of CamelCase, [src](https://github.com/justpy-org/justpy/blob/461a6419a728353c2fc59d24e8c495eed226abfd/justpy/justpy.py#L379), [pep](https://peps.python.org/pep-0008/#function-and-variable-names))
- `self.sleepTime=sleepTime` -> `self.sleep_time = sleepTime` 
(incorrect use of spaces around equal sign, [src](https://github.com/justpy-org/justpy/blob/461a6419a728353c2fc59d24e8c495eed226abfd/tests/base_server_test.py#L30), [pep](https://peps.python.org/pep-0008/#other-recommendations))
- `global jp_server,func_to_run, startup_func, HOST, PORT` -> `global jp_server, func_to_run, startup_func, HOST, PORT` 
(missing space after comma, [src](https://github.com/justpy-org/justpy/blob/461a6419a728353c2fc59d24e8c495eed226abfd/justpy/justpy.py#L401), [pep](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements))

Would suggest the use of the [black](https://github.com/psf/black) package to ascertain consistent code amongst all developers, in line with PEP8, while also appllying PEP8 more correctly w.r.t. CamelCase for classes, but lower_case for functions and variables.

**Comment by WolfgangFahl** [comment]
@platinops  could you please comment on #493 and be more specific on how we could apply black e.g by supplying a pull request.

**Comment by platinops** [comment]
Okey, done: #499

After this PR is merged, I can do another one for the inconsistent use of spaces, commas and capitals - if that's OK with you.
